externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - "9030"
 
   doris-be:
-    image: zhaomin1423/doris-be:1.0.0
+    image: zhaomin1423/doris-be:1.0.0-0.1
```

Comment:
could you provide the Dockerfile?

Reply:
add it to project?

Reply:
yes，it is helpful.

Reply:
yeah, it looks like a black box for Kyuubi if no more detials. We should add some comments to explain at least.

Reply:
added the image link included Dockerfiles to comment.